lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -169,18 +169,23 @@ def add_creditcard(post, payment, options)
         pm_fields[:expiration_month] = payment.month.to_s
         pm_fields[:expiration_year] = payment.year.to_s
         pm_fields[:name] = "#{payment.first_name} #{payment.last_name}"
-        pm_fields[:cvv] = payment.verification_value.to_s unless send_cvv_value?(payment, options)
+        pm_fields[:cvv] = payment.verification_value.to_s unless send_cvv_value?(payment, options) || valid_network_transaction_id?(options)
         add_stored_credential(post, options)
       end
 
       def send_cvv_value?(payment, options)
-        payment.verification_value.nil? || payment.verification_value&.empty? || (options[:stored_credential] && options[:stored_credential][:reason_type] == 'recurring')
+        return (payment.verification_value.nil? || payment.verification_value&.empty?)
```

Comment:
it means that not matter if is a recurring transaction ?

Reply:
Sort of, I think we can just simplify our condition to not be concerned with whether or not it's `recurring` since the main values we are checking for are `verification_value` and `network_transaction_id`.